c2cgeoportal/views/export.py

```diff
@@ -40,7 +40,7 @@
 
 
 @view_config(route_name="csvecho")
-def echo(request):
+def exportcsv(request):
```

Comment:
I missed that comment. Sorry.

I didn't want to change the ```route_name``` to avoid breaking existing applications that use `${request.route_url('csvecho')}` in mako templates.

Please tell me if you'd like to reconsider this.